### PR TITLE
Wiki entries

### PR DESCRIPTION
**Pull request by TheSlimvReal**
#198

### CODE REVIEW

File: wiki/Contributing_PyTest.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+> python -m pip install pytest
+
+or just automatically install all our dependencies
+
+> python -m pip -r install requirements.txt
+
+## Create a test
+
+First you create a class the is called something like `test_name.py`.
```
Comment (annaric) [inline wiki/Contributing_PyTest.md]: *that is called

File: wiki/Technologies.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+This is a list of the technologies and models we use for planing and developing this project.
```
Comment (annaric) [inline wiki/Technologies.md]: *planning

File: wiki/Contributing_PyTest.md (hunk 1 above)
Comment (TheSlimvReal) [reply wiki/Contributing_PyTest.md]: got fixed

File: wiki/Technologies.md (hunk 2 above)
Comment (TheSlimvReal) [reply wiki/Technologies.md]: planing is correct

**Pull request merged**